### ISSUE TITLE
Breadcrumbs is not working on collection filter links

### ISSUE THREAD

**Issue by lightbringer93** [statamic/cms#2987]
My breadcrumbs are not working on the collection pages.
I receive the following error: `Call to undefined method Statamic\Taxonomies\Taxonomy::setSupplement()`. The URL structure where I get the error is e.g. /publications/categories/annual-reports.
Publications is my collection and categories my taxonomy handle. 
I am just using the breadcrumbs navigation tag. On all other pages it works only on the filter pages its throwing this error. 
This the Stack trace. 
```
<!-- Breadcrumbs
============================================= -->
<nav class="hidden md:flex items-center w-full mb-4 px-4" aria-label="Breadcrumb">
    <ol class="flex items-center space-x-4">
        {{ nav:breadcrumbs }}
            <li>
                {{ partial:components/navigation/breadcrumb }}
            </li>
        {{ /nav:breadcrumbs }}
    </ol>
</nav>
```
This is my breadcrumb partial. 
```
<!-- Breadcrumb
============================================= -->
{{ if first }}
    <div>
        <a href="{{ url }}" class="text-gray-400 hover:text-gray-500">
            {{ svg src="/assets/icon/solid/home" class="flex-shrink-0 fill-current h-4 w-4" }}
            <span class="sr-only">{{ title }}</span>
        </a>
    </div>
{{ else }}
    <div class="flex items-center">
        {{ svg src="/assets/icon/regular/angle-right" class="flex-shrink-0 fill-current flex-shrink-0 h-3 w-3 text-gray-400" }}
        <a href="{{ url }}" class="ml-4 text-sm text-gray-500 hover:text-gray-700">
            {{ title }}
        </a>
    </div>
{{ /if }}
```
Do I have to bulid the breadcrumbs separately for these pages?
## Environment
Statamic 3.0.33 Pro
Laravel 8.18.1
PHP 7.4.11
No addons installed

**Comment by damcclean** [comment]
I've not tested it yet but I believe this bug should be fixed when 05601e4 is tagged.

**Comment by damcclean** [comment]
[v3.0.35 has been tagged](https://github.com/statamic/cms/releases/tag/v3.0.35) (with the commit I mentioned above). I'm closing this issue as I have a feeling that it will fix the issue you're running into here. 

If it's still happening after updating, then I can re-open the issue for you.
